packages/now-node/src/dev-server.ts

```diff
@@ -52,23 +52,72 @@ async function main() {
     config.helpers === false || process.env.NODEJS_HELPERS === '0'
   );
 
-  const entrypointPath = path.join(process.cwd(), entrypoint);
-  const handler = await import(entrypointPath);
+  bridge = getNowLauncher({
+    entrypointPath: join(process.cwd(), entrypoint),
+    helpersPath: './helpers',
+    shouldAddHelpers,
+    bridgePath: 'not used',
+    sourcemapSupportPath: 'not used',
```

Comment:
Something seems wrong here. Why `'not used'` instead of `undefined`?

Reply:
Cause TypeScript 🤦

Reply:
Seems like it should use a different type for this function then? I approved so we can move forward with this but I would like to revisit at some point.

Reply:
Agreed, sounds good.